includes/FeatureAPI/Permissions/WithdrawPermissions.php

```
@@ -0,0 +1,218 @@
+
+        return absint( $withdraw_user_id ) === get_current_user_id() && 
+               current_user_can( 'dokan_update_withdraw' );
+    }
+
+    /**
+     * Check if user can delete withdrawal
+     *
+     * @param array $context
+     * @return bool
+     */
+    public static function can_delete_withdrawal( $context ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$context'. (Unused Code Rules)

(UnusedFormalParameter)